tests/jobs/correlated_observations_scaling/unit/test_scaling_config.py

```diff
@@ -74,17 +70,16 @@ def test_config_value_overwrites_default(input_key):
         "CALCULATE_KEYS": {"std_cutoff": 1.0e-6, "alpha": 3},
         "UPDATE_KEYS": {},
     }
-    reporter = FileReporter(os.path.realpath(os.getcwd()))
-    job = ScalingJob(["a_key"], [], ["a_key"], user_config_dict, reporter, default_vals)
+    job = ObsCorrConfig(user_config_dict, [], default_vals)
```

Comment:
If we want to be picky, it's no longer a `job` instance :P

Reply:
Picky is good, confusing language is bad ;)